src/printer.js

```diff
@@ -1,4 +1,33 @@
 // https://prettier.io/docs/en/plugins.html#printers
-const genericPrint = (path, options) => options.originalText;
+const { concat, hardline, indent, join } = require('prettier').doc.builders;
+
+const space = ' ';
+
+const genericPrint = (path, options, print) => {
+  const node = path.getValue();
+
+  switch (node.type) {
+    case 'SourceUnit':
+      return join(hardline, path.map(print, 'children'));
+    case 'PragmaDirective':
+      return concat([
+        join(space, ['pragma', node.name, node.value]),
+        ';',
+        hardline
+      ]);
+    case 'ContractDefinition':
```

Comment:
I've added contract definition but I've got a very hard time w/ `hardline` and indentation ... not sure what's wrong w/ this case here 😢